jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceFactory.java

```diff
@@ -27,7 +27,88 @@
 import org.eclipse.jetty.util.component.Dumpable;
 
 /**
- * ResourceFactory.
+ * <p>ResourceFactory is the source of new {@link Resource} instances.</p>
+ *
+ * <p>
+ *     It is important to understand the scope of any {@code ResourceFactory}
+ *     objects you create, as all {@link Resource} instances created from
+ *     it will be tied to the life of the {@link ResourceFactory}.
```

Comment:
I think it would be better to introduce the fact that RF has a scope (I would not call it lifecycle).
After that you can say that resources created from it have the same scope.

What above reads: "the scope of any RF you create ... will be tied to the life of the RF" which is confusing as it is tautological.